plugin/path_config.go

```diff
@@ -90,6 +95,7 @@ func (b *backend) configUpdateOperation(ctx context.Context, req *logical.Reques
 	maxTTL := fieldData.Get("max_ttl").(int)
 	length := fieldData.Get("length").(int)
 	formatter := fieldData.Get("formatter").(string)
+	passwordLastSetBuffer := fieldData.Get("out_of_band_rotation_seconds").(int)
```

Comment:
Should probably check for >=0 (or maybe >0)?

Reply:
It should always be 1 because I set the field default as 1 at the field schema level.

Reply:
Double-checked and should be fine as-is because the `Get` function does call `GetDefaultOrZero` to populate the value from the default if it's set in the schema.

Reply:
Why I meant was that the user can currently do: `last_rotation_tolerance=-1h` and I think it will be accepted.